template-parts/content.php

```diff
@@ -15,7 +15,7 @@
 
 <article <?php post_class( 'post-container' ); ?>>
 
-	<header class="entry-header">
+	<header class="entry-header container">
```

Comment:
```suggestion
	<header class="entry-header">
```

Container is already applied.